### PR TITLE
more splashes!

### PR DESCRIPTION
**Pull request by NotAShelf**
> please double check if I added more than two songs per band/group/etc/whatever because I do not have the attention span to do so myself

#### Describe your PR, what does it fix/add?
Wish it fixed but depression, but no. It does not fix anything.
On the other hand, it adds (I think) 13 more music references to the splashes

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
see above

#### Is it ready for merging, or does it need work?
Good to go!

### CODE REVIEW

File: src/helpers/Splashes.hpp [hunk 1]
```diff
@@ -48,9 +48,22 @@ inline const std::vector<std::string> SPLASHES = {
...
+    "Wham, bam, shang a lang",
+    "Did you think I'd die? Oh, no, not I, I will survive.",
+    "But I would walk 500 miles, and I would walk 500 more",
+    "COUNTRY ROADS, TAKE ME HOME!",
+    "I was born, six gun in my hand",
+    "I am the storm that is aproaching",
+    "Daydream. I fell asleep amid the flowers",
+    "This ain't a scene, it's an arms race!",
+    "Sweet abyss come out tonight",
+    "I'm gonna be the anti-hero, feared and hated by everybody",
```
Comment (vaxerski) [inline src/helpers/Splashes.hpp:62]: no

File: src/helpers/Splashes.hpp [hunk 2]
```diff
@@ -48,9 +48,22 @@ inline const std::vector<std::string> SPLASHES = {
     "Ding ding pch n daa, bam-ba-ba-re-bam baram bom bom baba-bam-bam-bommm",
     "Súbeme la radio que esta es mi canción",
     "I'm beggin', beggin' you",
+    "People are strange, when you're a stranger",
+    "Sun is shining in the sky, there ain't a cloud in sight",
+    "Wham, bam, shang a lang",
+    "Did you think I'd die? Oh, no, not I, I will survive.",
+    "But I would walk 500 miles, and I would walk 500 more",
+    "COUNTRY ROADS, TAKE ME HOME!",
```
Comment (vaxerski) [inline src/helpers/Splashes.hpp:56]: no caps pls

**Comment by ThatOneCalculator** [comment]
Ngl these all kinda suck :/

**Comment by NotAShelf** [comment]
1. that's rude
2. you do realise there's an entire section of song lyrics that are not remotely funny, relevant or Hyprland related. I am merely contributing more song lyrics.

**Comment by NotAShelf** [comment]
> I have heard literally nobody ever reference that song tbh

there are 13, which one are you referring to

**Pull request closed without merging**